index.html

```diff
@@ -1,14 +1,239 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
+
+<head>
     <title>Mockup Home</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <section id="header">
+        <h1>The WORST PAGE EVER!</h1>
+        <h3>The Three Amigos!</h3>
+
+    </section>
+    <section id="features" class="content-width">
+        <div id="page-break-1">
+
+        </div>
+        <div class="title">
+        </div>
+        <div id="home-img-container">
+            <div class="home-image-wrapper">
+                <img class="image" src="https://www.fillmurray.com/200/300">
+            </div>
+
+            <div class="home-image-wrapper">
+                <img class="image" src="https://www.placecage.com/g/200/300">
+            </div>
+
+            <div class="home-image-wrapper">
+                <img class="image" src="https://www.stevensegallery.com/200/300">
+            </div>
+        </div>
+        <button class="purple-btn" style="text-align: center" ;>Click Here!</button>
+    </section>
+
+
+    <section id="main">
+        <div class="content">
+
+            <div class="post">
+                <h3 class="post-title">ES-6 </h3>
+                <div class="post-img-wrapper">
+
+                </div>
+
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Et deserunt nulla tempora consectetur
+                    autem incidunt, voluptate optio aspernatur modi iste explicabo, debitis ad magnam repellendus
+                    nihil eos sed adipisci magni sunt ipsa maiores pariatur necessitatibus! Quod voluptates
+                    praesentium adipisci? Nostrum, ullam dicta magnam sit accusamus recusandae, nihil sequi
+                    temporibus sed error optio. Ea, maiores molestiae. Reiciendis mollitia fugit, placeat
+                    praesentium ducimus, deserunt accusamus velit consectetur aliquid magni excepturi odio! Labore!
+                </p>
+                <form action="blog-post.html">
+                    <button class="dark-gray-btn">Read On!</button>
+                </form>

```

Comment:
This is an example of where you could use an `<a>` tag rather then a button for navigation